### ISSUE TITLE
Does only support SSLv2&3

### ISSUE THREAD

**Issue by kelunik** [PeeHaa/Requestable#25]
It seems like it does only support SSLv2&3 currently. Both shouldn't be used any longer and disabled by servers if possible.

https://requestable.pieterhordijk.com/EFCuvc fails with a SSL connect error. However, the following works: https://requestable.pieterhordijk.com/GFCuvc

According to https://www.ssllabs.com/ssltest/analyze.html?d=https%3A%2F%2Frequestable.pieterhordijk.com, SSLv2&3 are disabled here, too.

**Comment by PeeHaa** [comment]
TLS 1.0, 1.1 and 1.2 are merged into master. However for some reason ampchat still fails load...

The only things I can think of are:
- I fucked something up (again)
- Your TLS setup does something cURL doesn't like
- I fucked up

http://stackoverflow.com/q/28419197/508666

**Comment by kelunik** [comment]
I changed the suites to `Intermediate` and it works now. ;-)
See https://wiki.mozilla.org/Security/Server_Side_TLS

**Comment by rdlowrey** [comment]
lol you had to allow weaksauce cipher suites so curl could connect :)

**Comment by PeeHaa** [comment]
@kelunik I will recompile curl on my machine later so that everything just works as it should.

Also thanks for the configuration chart. /cc @daverandom (if you are still alive)

**Comment by kelunik** [comment]
Anyway, I'll probably leave it at `Intermediate`, because IE 10 is theoretically in our support range, last two majors.

**Comment by PeeHaa** [comment]
Yeah agreed. For now I would leave it on `Intermediate` for the project. And by the time stupid user agents died I am confident I have recompiled curl on my machine :)

**Comment by kelunik** [comment]
I think IE 10 support will die anyway when Windows 10 is released with its new browser, because IE 10 still needs the 2012 syntax: http://caniuse.com/#search=flex :-)
